src/avatar.scss

```
@@ -49,6 +49,12 @@ $block: #{$fd-namespace}-avatar;
   background-position: 50%;
   border-radius: var(--sapElement_BorderCornerRadius);
 
+  @include fd-focus() {
+    outline-width: var(--sapContent_FocusWidth);
+    outline-color: var(--sapContent_FocusColor);
+    outline-style: var(--sapContent_FocusStyle);
```

```suggestion
  @include fd-fiori-focus(0);
```
please use fd-fiori-focus instead of fd-focus

---

Hi Yuri,

It uses _outline-offset_ which is not supported in IE11, to avoid this issue used fd-focus.